### PR TITLE
Documents

### PR DESCRIPTION
**Pull request by TransportSolutions**

### CODE REVIEW

File: server/routes/index.js [hunk 1]
```diff
@@ -1,7 +1,22 @@
 let express = require('express'),
   controllers = require('../controllers/all'),
-  appControllers = require('../controllers/app/all');
-adminControllers = require('../controllers/admin/all');
+  appControllers = require('../controllers/app/all'),
+  multer = require('multer'),
```
Comment (gonzalolanderos) [inline server/routes/index.js]: What's this for?

Comment (TransportSolutions) [inline server/routes/index.js]: multer is express middleware to upload documents to disk.

Comment (TransportSolutions) [inline server/routes/index.js]: I have implemented document upload and fetch here but I think we need one forms module also, where user can open, fill form and send it to anyone.

Comment (gonzalolanderos) [inline server/routes/index.js]: Ok, looks good. Forgot to merge :)

**Pull request merged**